explore/src/main/scala/explore/observationtree/AsterismGroupObsList.scala

```diff
@@ -292,10 +327,37 @@ object AsterismGroupObsList {
               "Asterism Summary"
             )
           ),
-          <.div(ExploreStyles.ObsTree)(
-            <.div(ExploreStyles.ObsScrollTree)(
-              asterismGroups.toTagMod(renderGroup)
-            )
+          ReflexContainer()(
+            List[TagMod](
+              ReflexElement(minSize = 36, clazz = ExploreStyles.ObsTreeSection)(
+                Header(block = true, clazz = ExploreStyles.ObsTreeHeader)("Asterisms"),
+                <.div(ExploreStyles.ObsTree)(
+                  <.div(ExploreStyles.ObsScrollTree)(
+                    asterismGroups.toTagMod(renderAsterismGroup)
+                  )
+                )
+              ),
+              ReflexSplitter(propagate = true),
+              ReflexElement(minSize = 36, clazz = ExploreStyles.ObsTreeSection, withHandle = true)(
```

Comment:
Is `minSize` something that can be in css or is it a param used by the component

Reply:
Heh. I don't know. I just resurrected it from the old targets tab, but I'll take a look.

Reply:
It is a parameter to the JS component.